### PR TITLE
Update add-or-remove-members-from-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details on which admin roles are needed to make the changes prescribed in the doc

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/860

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/create-groups/add-or-remove-members-from-groups.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ description: "Learn how to add a member to a group, remove member from group, an
...
+> If you're not using the new Microsoft 365 admin center, you can turn it on by selecting the **Try the new admin center** toggle located at the top of the Home page. 
 
 ::: moniker-end
 
-In Office 365, Group members typically create their own Groups, add themselves to Groups they want to join, or are invited by Group owners. If Group ownership changes, or if you determine that a member should be added or removed, as the admin you can also make that change. [What is an Office 365 Group?](https://support.office.com/article/b565caa1-5c40-40ef-9915-60fdb2d97fa2.aspx)
+In Office 365, Group members typically create their own Groups, add themselves to Groups they want to join, or are invited by Group owners. If Group ownership changes, or if you determine that a member should be added or removed, as the admin you can also make that change. Only a Global Administrator, Exchange adminsitrator, Groups adminstrator or User administrator can make these changes. [What is an Office 365 Group?](https://support.office.com/article/b565caa1-5c40-40ef-9915-60fdb2d97fa2.aspx)
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/add-or-remove-members-from-groups.md]:
```suggestion
In Office 365, Group members typically create their own Groups, add themselves to Groups they want to join, or are invited by Group owners. If Group ownership changes, or if you determine that a member should be added or removed, as the admin you can also make that change. Only a global administrator, Exchange administrator, Groups administrator, or user administrator can make these changes. [What is an Office 365 Group?](https://support.office.com/article/b565caa1-5c40-40ef-9915-60fdb2d97fa2.aspx)
```

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #860.

Thank you.

cc: @kenwith

**Pull request merged**